jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/http/DslHttpSampler.java

```diff
@@ -260,7 +300,16 @@ public DslHttpSampler clientImpl(HttpClientImpl clientImpl) {
   @Override
   public TestElement buildTestElement() {
     HTTPSamplerProxy ret = new HTTPSamplerProxy();
-    // this might be null if the url is set by defaults element
+    // this might be null if the values are set by defaults element
```

Comment:
they can also be null if url has the full path. It may even make sense to rename existing url to path.

Reply:
you right, rename it